### DISCUSSION TITLE
''Go to source'' command missing from pdf viewer

### DISCUSSION THREAD

**Discussion by PhilScience** [texstudio-org/texstudio#1914] [Q&A] [answered]
I've posted this in the ''Issues'' section but didn't receive any answers, could someone please help me to resolve this issue? I'm writing my thesis in LaTex and this is quite annoying...

Environment
TeXstudio: <TeXstudio 4.0.2 (git 4.0.2)>
Qt: < Using Qt Version 6.2.0, compiled with Qt 6.2.0 R >
OS: < Windows 10 and Windows 11>
TeX distribution: < miktex >

Expected behavior
When building a document, the drop menu when right clicking on the pdf viewer begins with the "Go to source command''. Also, the mouse pointer is typically an arrow, unless 'scroll' is pressed which turns the mouse pointer into a hand.

Actual behavior
The ''Go to source'' command keeps disappearing from the drop down menu. When I open a new project and compile a new document it reappears on the pdf viewer, but after a couple of ''built & view'''s it disappears again. Also, the mouse pointer has permanently switched into a hand (scroll) and will not return to an arrow, no matter what I do. I 've tried to restore the original settings by removing the .ini file, and I have also completely uninstalled and re-installed TexStudio on my laptop but the behavior remains the same for some reason!

How to reproduce
I have no idea, sorry.

**Comment by muzimuzhi** [comment]
When the "Go to source" command disappears, does the `<filename>.synctex.gz` file exist?

**Reply by muzimuzhi** [reply]
hmm, I don't think my words would account for an answer.

**Reply by PhilScience** [reply]
Yes, it does. I've also tried to delete all the auxiliary files and compile again, but nothing changes.

**Reply by sunderme** [reply]
The problem is that if the issue is not reproducible, how should that be debugged ?

**Reply by muzimuzhi** [reply]
If the `.synctex.gz` exists and locates at the same directory of your root `.tex` file, then you can upload a zipped file containing (a simplified or even minimal) `.tex`, `.pdf`, and `.synctex.gz` which can reproduce the problem on your side.

**Comment by sunderme** [comment, marked as answer]
please, discuss bug report on the issues.
That being said, if there is not a way to reproduce the issue, there is nothing I can do.
